hs-ucan/test/Test/Prelude.hs

```diff
@@ -1,4 +1,4 @@
-module Test.Web.UCAN.Prelude
+module Test.Prelude
```

Comment:
I would scope this down. `Test.Prelude` is just so general that a namespace collision feels likely. I also see a lot of stuff like `Test.AWS.Prelude` in the wild (https://www.stackage.org/haddock/lts-18.23/amazonka-test-1.6.1/Test-AWS-Prelude.html)